segmentation_models/models/_utils.py

```diff
@@ -8,3 +8,9 @@ def freeze_model(model, **kwargs):
         if not isinstance(layer, layers.BatchNormalization):
             layer.trainable = False
     return
+
+
+def get_submodules_args(kwargs):
```

Comment:
Lets call it `filter_kwargs` or smth like that, so it woulf be more clear?
It is kind of messy to have `get_submodules_from_kwargs` and `get_submodules_args`

Reply:
Is renaming ```get_submodules_args``` to ```filter_keras_submodules``` enough?